apis_core/apis_entities/abc.py

```diff
@@ -21,6 +24,24 @@ class E21_Person(LabelBaseModel, models.Model):
     class Meta:
         abstract = True
 
+    @property
+    def node(self):
+        if self.uri_set.all():
+            return URIRef(self.uri_set.first().uri)
+        return BNode()
+
+    def triple(self):
+        return (self.node, RDF.type, FOAF.Person)
```

Comment:
We can not use FOAF for the serialization and call the entity `E21_Person` which is a cidoc-crm class. We should consistently use one or the other